connect/transforms/src/main/java/org/apache/kafka/connect/transforms/RegexRouter.java

```diff
@@ -20,12 +20,15 @@
 import org.apache.kafka.connect.connector.ConnectRecord;
 import org.apache.kafka.connect.transforms.util.RegexValidator;
 import org.apache.kafka.connect.transforms.util.SimpleConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class RegexRouter<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger log = LoggerFactory.getLogger(RegexRouter.class);
```

Comment:
I'm not sure any of these log messages add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.